src/components/index-page/css/index-hero.css

```diff
@@ -0,0 +1,34 @@
+
+.index-hero {
+  @apply flex justify-center w-full xl:py-16 py-4;
+  @apply bg-white relative z-30;
+}
+
+.index-hero__contents {
+  @apply flex flex-col justify-between items-center w-full max-w-7xl xl:flex-row xl:px-10;
+}
+
+.index-hero__text-container {
+  @apply flex flex-col xl:items-start px-4 pt-8 xl:p-0;
+  @apply xl:text-left text-center;
+}
+
+.index-hero__heading {
+  @apply text-3xl mb-6 px-6 xl:px-0;
+}
+
+.index-hero__text {
+  @apply text-lg px-6 xl:px-0;
+}
+
+.index-hero__image-container {
+  @apply px-4 py-8 xl:p-0;
+}
+
+.index-hero__image {
+  @apply rounded-3xl;
```

Comment:
if this treatment is repeated throughout we might want to either create a `<RoundedImage/>` component or write a `@layer component` for this behavior